bin/Modules/Modules/EquipmentModule.js

```diff
@@ -53,13 +53,14 @@ class EquipmentModule extends GModule {
                     if (await Globals.connectedUsers[res.locals.id].character.haveThisObject(toEquip)) {
                         if (await Globals.connectedUsers[res.locals.id].character.getInv().isEquipable(toEquip)) {
                             let tItemToEquip = await Globals.connectedUsers[res.locals.id].character.getInv().getItem(toEquip);
+                            let tItemToEquipName = tItemToEquip.getName(data.lang);
                             if (Globals.connectedUsers[res.locals.id].character.getLevel() >= tItemToEquip.getLevel()) {
                                 let swapItem = await Globals.connectedUsers[res.locals.id].character.getEquipement().equip(tItemToEquip.id);
                                 await Globals.connectedUsers[res.locals.id].character.getInv().deleteFromInventory(tItemToEquip);
                                 if (swapItem > 0) {
                                     await Globals.connectedUsers[res.locals.id].character.getInv().addToInventory(swapItem);
                                 }
-                                data.success = Translator.getString(res.locals.lang, "inventory_equipment", "item_equiped");
+                                data.success = Translator.getString(res.locals.lang, "inventory_equipment", "item_equiped") + " (" + tItemToEquipName + ")";
```

Comment:
Modify "inventory_equipment", "item_equiped" with (%s) instead of adding it after the string

Reply:
reee so many nitpicks
how dare you expose my code for being lazy if it works
I'll try to do them